client-app/pages/matcher/matcher.vue

```diff
@@ -3,12 +3,14 @@
     <SlugContent
       v-if="previewers.slugContent.isActive"
       v-show="visibleComponent === 'slugContent'"
+      :is-visible="visibleComponent === 'slugContent'"
```

Comment:
Before we could having 2 `categories.vue` components on the page.
Since `useProducts` is a singleton, it's properties are shared between 2 components.
That leads to bugs. For example in Builde IO we had set `fixedProductsCount` to `8`, but got 16 because of the second component (default category view with defaulp product qty)
So I pass `isVisible` prop to hide inner components.